orm.go

```diff
@@ -709,6 +693,31 @@ func createAttributesString(attrs map[string]interface{}) (string, error) {
 	return strings.Join(attrsList, ","), nil
 }
 
+func formatIDKey(idKey interface{}) (string, error) {
+	switch v := idKey.(type) {
+	case uint64:
+		return strconv.FormatUint(v, 10), nil
+	case string:
+		return fmt.Sprintf(`'%s'`, v), nil
+	case int:
+		return strconv.FormatInt(int64(v), 10), nil
+	default:
+		return "", errors.New("Uknown type for id/key")
```

Comment:
errors should not start with a capital letter, and unknown is typo'd.  Let's also include the errant value, so

`errors.Errorf("id/key is not a string or integer type: %#v", idKey)`